test/integration/consul-container/test/gateways/gateway_endpoint_test.go

```diff
@@ -183,27 +224,28 @@ func createServices(t *testing.T, cluster *libcluster.Cluster, ports ...int) (li
 	return clientConnectProxy, gatewayService
 }
 
-// checkRoute, customized version of libassert.RouteEchos to allow for headers/distinguishing between the server instances
-
 type checkOptions struct {
 	debug      bool
 	statusCode int
 	testName   string
 }
 
-func checkRoute(t *testing.T, ip string, port int, path string, headers map[string]string, expected checkOptions) {
+// checkRoute, customized version of libassert.RouteEchos to allow for headers/distinguishing between the server instances
+func checkRoute(t *testing.T, port int, route string, headers map[string]string, expected checkOptions) {
```

Comment:
#nit would continue using `path` as the variable name here since it's clearer and less overloaded among config entry names and whatnot

Reply:
Agreed, looks like I just missed updating it here.